api/projects/tags/models.py

```diff
@@ -12,6 +16,22 @@ class Tag(AbstractBaseExportableModel):
     description = models.CharField(max_length=512, blank=True, null=True)
     project = models.ForeignKey(Project, on_delete=models.CASCADE, related_name="tags")
 
+    is_system_tag = models.BooleanField(
+        default=False,
+        help_text="Indicates that a tag was created by the system, not the user.",
+    )
+    is_permanent = models.BooleanField(
+        default=False,
+        help_text="When applied to a feature, it means this feature should be excluded from stale flags logic.",
+    )
+    type = models.CharField(
+        default=None,
+        null=True,
```

Comment:
I'd prefer to have a default tag type.

Reply:
You mean like `TagType.NONE`/

Reply:
Yes, or something among the lines of `TagType.NORMAL`.

Reply:
I went with TagType.NONE as I feel that is the most relevant here. It also means that I've had to complicate the migration a little bit to create the nullable column, populate the data, then add the default and remove the null option.